torchvision/prototype/features/_label.py

```diff
@@ -43,10 +43,10 @@ def from_category(
         return cls(categories.index(category), categories=categories, **kwargs)
 
     def to_categories(self) -> Any:
-        if not self.categories:
-            raise RuntimeError()
+        if self.categories is None:
+            raise RuntimeError("Label does not have categories")
 
-        return apply_recursively(lambda idx: cast(Sequence[str], self.categories)[idx], self.tolist())
+        return tree_map(lambda idx: self.categories[idx], self.tolist())
```

Comment:
We have a lot of calls to `apply_recursively`, should we update them all?

Reply:
but also, it looks private - should we actually use it ?

Reply:
Yes, in transforms I already replaced them.